tests/test_vjp.py

```diff
@@ -108,6 +108,18 @@ def test_zero_dy(self, dev, C):
         vjp = dev.compute_vjp(dy, jac)
         assert np.all(vjp == np.zeros([3]))
 
+    @pytest.mark.parametrize("C", [np.complex64, np.complex128])
```

Comment:
I have added tests for non_numpy `dy` as part of #239. You may want to remove this here.

Reply:
Aha! Yes, as your version tests all interfaces, it's more reasonable to remove this here.